en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -15,12 +15,13 @@ We'll go through each of these in detail in the following sections.
 | Main interface | Function |
 | :--- | :--- |
 | Power | Connect Power module;  power input & AD voltage  and current detection. |
-| PM2 | Connect i2c smart battery. |
-| TF CARD | Insert SD card for log storage |
+| PM2 | [Do not use with PX4](../flight_controller/cuav_v5_nano.md#issue_pm2) |
+| TF CARD | SD card for log storage (comes with card) |
 | M1~M8 | PWM outputs. Can be used to control motors or servos. |
-| A1~A8 | PWM outputs. Can be used to control motors or servos. |
+| A1~A3 | PWM outputs. Can be used to control motors or servos. |
+| nARMED | |
 | DSU7 | Used for FMU debug, reading debug information. |
-| I2C1/I2C2 | Connect an I2C device such as an external compass. |
+| I2C1/I2C2/I2C3/I2C4 | Connect an I2C device such as an external compass. |
```

Comment:
I see nano has I2C3 and I2C4. But I can't see I2C1. 
- Is there an I2C1?
- Is the description OK for "all of these" - ie do we need to mention other uses
- Is there anything else we need to say about the I2C ports/bus?

Reply:
Sorry, this is wrong, v5 nano does not have an i2c1 interface.

Reply:
Thanks @cuhome! I will remove I2C1. 

What about the other questions:
- Do we need to mention anything other than I2C compasses or is this good enough?
- Is there anything else we need to say about the I2C ports/bus?

Reply:
Great,no need to add other descriptions